x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/examples_list/examples_list.tsx

```diff
@@ -22,7 +20,14 @@ export const ExamplesList: FC<Props> = ({ examples }) => {
   }
 
   const examplesContent = examples.map((example, i) => {
-    return <Example key={`example_${i}`} example={example} />;
+    return (
+      <EuiListGroupItem
+        style={{ padding: 0 }}
+        size="xs"
+        key={`example_${i}`}
+        label={typeof example === 'string' ? example : JSON.stringify(example)}
```

Comment:
looking at the response from `get_field_stats` is there a risk here that an example could be a number or null?
which this check would attempt to stringyfy.
it might be safer to switch this around and check for `object`

Reply:
I just copied this condition from the `Example` component I deleted

Reply:
ok, must be fine then.